### ISSUE TITLE
Automatic API documentation generation

### ISSUE THREAD

**Issue by ricardoboss** [bytedance/unpub#87]
Hi! Will unpub support automatically generating API documentation from source code, like pub.dev does?

**Comment by talisk** [comment]
Hi @ricardoboss !
Did you find the documentation like this?
https://pub.dev/documentation/unpub/latest/

**Comment by ricardoboss** [comment]
No, I meant automatic package API documentation for packages uploaded to unpub.
